ddtrace/tracer.py

```diff
@@ -632,6 +632,7 @@ def _start_span(
             else:
                 service = config.service
 
+        # DEV: We *MUST* use `mapped_service` from here on
```

Comment:
If we don't need the original service name beyond this point, can we just do
~~~python
service= config.service_mapping.get(service, service)
~~~
?

Reply:
I did think about this too, I was hoping someone would call it out.

This is probably the more clear change. Thanks!